test/CppShared/TestDynamicDependency.cpp

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+
+using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+
+namespace ProjectReunionCppTest
+{
+    TEST_CLASS(DynamicDependenciesTest)
```

Comment:
?

Reply:
Leftover from an early start that didn't pan out. Didn't notice the file was still around as I'm living the VS dream and it's not in the VS project view.

Something we should keep in mind for later. Some kind of "xref filesystem vs .vcxproj and flag any files in the former not in the latter" tool at least. Would be useful to help spot garbage accidentally laying around. @EHO-makai do you have a running list of 'build process checks' we want to eventually have? Or a consistent place we can log such things for future action?

Reply:
Added issue #179 Add 'scan for unreferenced files' tool